projects/ngx-d3timeline/src/lib/axis/axis.ts

```diff
@@ -4,5 +4,6 @@ import { Line } from '../core/line';
 export interface Axis {
   tickMarks: TickMark[];
   line: Line;
+  showGridLines: boolean;
```

Comment:
I considered leaving this off but not generating the grid lines further up if it was false. Currently its a bit inefficient but maybe more definitive. What do you think?